docs/usage/management-api-walkthrough/02_policies.md

```diff
@@ -15,8 +39,188 @@ Content-Type: application/json
+geography wouldn't even have them in their catalog.
+- **usage policy or contract policy:** determines the conditions for initiating a contract negotiation for a particular 
+asset. Note that does not automatically guarantee the successful creation of a contract, it merely expresses the 
+eligibility to start the negotiation. The terms "usage policy" and "contract policy" are used synonymously!
+
+**The Access and Usage Policies are not distinguished by any special semantics, but rather by the time at which they are 
+checked.**
+
+Whether a policy is used as access or usage policy is determined during [contract definition](03_contractdefinitions.md).
+
+### Exemplary scenarios
+
+For the following Scenarios, we assume there is a **Partner 1 (provider)** who wants to provide Data for **Partner 2 
+(consumer)**
+
+- Partner 1 (provider) has the Business-Partner-Number BPN12345.
+- Partner 2 (consumer) has the Business-Partner-Number BPN6789.
+
+Partner 2 (consumer) signed the **Traceability Framework Agreement** and followed all the necessary steps that the 
+Credential appears within Partner 2s identity.
+
+So for this example, if Partner 2 does a catalog request to Partner 1, the following credential is provided to 
+Partner 1:
+
+```json
+{
+  "@context": [
+    "https://w3id.org/2023/catenax/credentials/summary/v1"
+  ],
+  "id": "<credential uid>",
+  "type": [
+    "VerifiableCredential",
+    "SummaryCredential"
+  ],
+  "issuer": "<did:web:issuer>",
+  "issuanceDate": "2023-06-02T12:00:00Z",
+  "expirationDate": "2022-06-16T18:56:59Z",
+  "credentialSubject": {
+    "id": "<did:web:subject>",
+    "holderIdentifier": "BPN6789",
+    "items": [
+      "MembershipCredential",
+      "TraceabilityCredential"
+    ],
+    "contractTemplates": "https://public.catena-x.org/contracts/"
+  }
+}
+```
+> Note: This specific credential schema will not be supported in upcoming releases but serves the purpose to illustrate
+> what such a credential may look like.
+
+#### Scenario 1
+
+Partner 1 wants to create an Access Policy, that Partner 2 can only receive the Contract offer if its BPN matches AND if
+it has the Traceability Credential (Traceability Contract Signed). If one of those is missing, Partner 2 won't receive a 
+Contract Offer.
+
+##### Partner 1 - Access Policy Example
+
+```json
+    {
+  "@context": {
+    "odrl": "http://www.w3.org/ns/odrl/2/"
+  },
+  "@type": "PolicyDefinitionRequest",
+  "@id": "{{POLICY_ID}}",
+  "policy": {
+    "@type": "Policy",
+    "odrl:permission": [
+      {
+        "odrl:action": "use",
+        "odrl:constraint": {
+          "@type": "LogicalConstraint",
+          "odrl:and": [
+            {
+              "@type": "Constraint",
+              "odrl:leftOperand": "BusinessPartnerNumber",
+              "odrl:operator": {
+                "@id": "odrl:eq"
+              },
+              "odrl:rightOperand": "{{BPN6789}}"
```

Comment:
why the bpn number is wrapped by double curly braces?

Reply:
Looks like a leftover from a postman variable :-)